plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -8745,6 +8841,11 @@ private String getTableLocation(String tableName)
         return (String) computeScalar("SELECT DISTINCT regexp_replace(\"$path\", '/[^/]*$', '') FROM " + tableName);
     }
 
+    private String getPartitionedTableLocation(String tableName)
```

Comment:
Adapt the code to look a bit more like https://github.com/trinodb/trino/blob/f2b1cfc4b601ce7c7d0bd518de546d1606025aa3/testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/util/TableLocationUtils.java#L33-L52